history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -1642,24 +1680,34 @@ this is that saving word of grace which I commend you unto;
 and that God may give you a heart inclinable to believe and obey the truth,
 is the desire of your faithful friend,
 
-R+++.+++ BARCLAY.
+[.signed-section-signature]
+R+++.+++ Barclay.
 
+[.signed-section-context-close]
 From the prison of Aberdeen, in Scotland,
 where I am confined for the sake of the testimony of Jesus.
-November 24,1676.
+
+[.signed-section-context-close]
+November 24, 1676.
 
 This letter, a year ago,
 at the desire of my friend R. B. I delivered into the hands of the afore-named ambassador,
 desiring his answer in writing, which he then promised; but not having as yet done,
 it was seen fit to be published.
 
-B.F. Rotterdam, the 28th of March, 1678.
+[.signed-section-signature]
+B+++.+++F.
+
+[.signed-section-context-close]
+Rotterdam, the 28th of March, 1678.
+
+--
 
-A brief and distinct Solution of the Argument which the
-Ambassador aforesaid uses against Robert Barclay`'s Thesis,
-whereby he attempts to evince that not the inward revelation of the Holy Spirit,
-but the outward by the Scripture, is the principal rule, and foundation of our faith:
-at least to us Europeans, who have the Scriptures.
+[.blurb]
+=== A brief and distinct Solution of the Argument which the Ambassador aforesaid uses against Robert Barclay`'s Thesis, whereby he attempts to evince that not the inward revelation of the Holy Spirit, but the outward by the Scripture, is the principal rule, and foundation of our faith: at least to us Europeans, who have the Scriptures.
+
+[.embedded-content-document.letter]
```

Comment:
I wasn't sure if I should embed this under a the [.blurb] ( or even if the [.blurb] was right!!) but it has the signatures at the bottom so I reasoned to embed it.

Reply:
I think you did right to embed it, and the blurb seems fine -- it's such a long title.  The one thing I think you should change is moving the headline to be _inside_ the embedded doc, then we'll call it good